app/components/post_list/post/avatar/avatar.tsx

```diff
@@ -39,12 +40,15 @@ const Avatar = ({author, enablePostIconOverride, isAutoReponse, location, post}:
 
     const fromWebHook = post.props?.from_webhook === 'true';
     const iconOverride = enablePostIconOverride && post.props?.use_user_icon !== 'true';
+    const propsIconUrl = ensureString(post.props?.override_icon_url);
+    const propsUsername = ensureString(post.props?.override_username);
```

Comment:
There's a slight change with these because it's valid for `override_icon_url` and `override_username` to be undefined rather than empty, and `ensureString` will turn these to be the empty string instead of `undefined`. Is there any chance that that would cause issues where these are passed to?